gobblin-yarn/src/main/java/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -545,6 +545,8 @@ public static void main(String[] args) throws Exception {
       Log4jConfigurationHelper.updateLog4jConfiguration(
           GobblinApplicationMaster.class, Log4jConfigurationHelper.LOG4J_CONFIGURATION_FILE_NAME);
 
+      LOGGER.info(YarnHelixUtils.getJvmInputArguments());
```

Comment:
At the moment the purpose is just so I can debug, profile, look at jxm, etc.  In the future I would like this information to be available somewhere via the admin api & ui.  It feels a little out of scope for me to do this now.